README.md

```diff
@@ -1,4 +1,99 @@
 Sailjail permissions
 ====================
-
 Permissions for [Sailjail](https://github.com/sailfishos/sailjail/).
+These are [firejail](https://firejail.wordpress.com/) profile files with
```

Comment:
Should this link to the webpage, upstream repository or our packaging of firejail?

Reply:
As long as upstream is identified and can be found, I think it would be better to refer to repos that we use for sailfish.